types/hummus-recipe/hummus-recipe-tests.ts

```diff
@@ -1,4 +1,4 @@
-import { Recipe } from 'hummus-recipe';
+import * as Recipe from "hummus-recipe";
```

Comment:
Since this is a callable or constructable entity, it isn't legal in ES2015 to import and call/construct this with a namespace import like `import * as foo from 'foo'`, so could you please convert this to either

```ts
import foo = require('foo')
```

or the following with `esModuleInterop` turned on

```ts
import foo from 'foo';
```

Reply:
Thank you for pointing it out